tony-core/src/test/java/com/linkedin/tony/TestTonyE2E.java

```diff
@@ -251,4 +251,26 @@ public void testTonyResourcesFlag() throws ParseException {
     int exitCode = client.start();
     Assert.assertEquals(exitCode, 0);
   }
+
+
+  /*
+   * In conditional_wait.py script, we wait for 5s for chief job type. If we only wait for worker to finish, this test
+   * is gonna fail.
+   */
+  @Test
+  public void testWaitingForChiefToFinish() throws ParseException {
+    conf.setBoolean(TonyConfigurationKeys.IS_SINGLE_NODE, false);
+    conf.setInt(TonyConfigurationKeys.TASK_MAX_MISSED_HEARTBEATS, 2);
+    client = new TonyClient(conf);
+    client.init(new String[]{
+        "--src_dir", "tony-core/src/test/resources/",
+        "--executes", "tony-core/src/test/resources/conditional_wait.py",
```

Comment:
After https://github.com/linkedin/TonY/pull/131, we should be able to just specify `--executes conditional_wait.py`, no?

Reply:
Yah, i branched off master for this fix. didn't pick up #131 . will rebase that.